lib/api/node.go

```diff
@@ -65,6 +65,17 @@ type NodeSpec struct {
 	// BGP configuration for this node.  If this omitted, the Calico node
 	// will be run in policy-only mode.
 	BGP *NodeBGPSpec `json:"bgp,omitempty" validate:"omitempty"`
+
+	// OrchRefs for this node.
+	OrchRefs []OrchRef `json:"orchRefs,omitempty" validate:"omitempty"`
+}
+
+// OrchRef is used to map a node in etcd to nodes in other orchestrators.
```

Comment:
nit: this shouldn't say anything about etcd - this is the Calico API structures and they don't care about what datastore is in use.

So, this should say something like "used to correlate a Calico node to its corresponding representation in a given orchestrator"